src/auth/authconfig/jwt.config.ts

```
@@ -0,0 +1,13 @@
+import { registerAs } from '@nestjs/config';
+
+export default registerAs('jwt', () => {
+  return {
+    secret: process.env.JWT_SECRET,
+    audience: process.env.JWT_TOKEN_AUDIENCE,
+    issuer: process.env.JWT_TOKEN_ISSUER,
+    ttl: parseInt(process.env.JWT_ACCESS_TOKEN_TTL ?? '3600'),
+    refreshttl: parseInt(process.env.JWT_REFRESH_TOKEN_TTL ?? '7776000'),
+    googleClient_id: process.env.GOOGLE_OAUTH_CLIENT_ID,
+    googleClient_secret: process.env.GOOGLE_OAUTH_CLEINT_SECRET
```

ESLint

[error] Insert `,`

(prettier/prettier)